### ISSUE TITLE
Update the screenshots in the README and github.io page

### ISSUE THREAD

**Issue by cxw42** [cxw42/TabFern#187]
The ones we currently have are over a year old!  Make some new screenshots.  Bonus points for including demoscene web sites in your list :) .

The README is in the `master` branch.  The github.io site is in the [gh-pages branch](https://github.com/cxw42/TabFern/tree/gh-pages).  It uses Jekyll as the site generator --- see [GitHub's docs](https://help.github.com/en/articles/using-jekyll-as-a-static-site-generator-with-github-pages).

Please do not include any confidential information or mature-audiences material in any screenshot!  I will regretfully decline any PRs including any of that type of content, or other content I think might reflect poorly on TabFern.  Thank you for your assistance!

**Comment by Prateek93a** [comment]
@cxw42 So only the TabFern extension image and the backup and restore image has to be changed?

**Comment by cxw42** [comment]
@Prateek93a Yes, at minimum.  If you are able to make some larger screenshots for the Chrome Web Store page, that would be a bonus!

**Comment by Prateek93a** [comment]
@cxw42 Can I take up this issue for the Hacktoberfest contest?

**Comment by cxw42** [comment]
Sure - I figured that was why you were asking :D .  Thanks!

**Comment by Prateek93a** [comment]
Ok 😅

**Comment by cxw42** [comment]
Please see [updated contribution guidelines](https://github.com/cxw42/TabFern/blob/master/CONTRIBUTING.md#Legal).  Thanks!

**Comment by cxw42** [comment]
@Prateek93a I'd still be interested in accepting your contribution if you are willing!  If not, please let me know so I can un-assign you.  Thanks!
